ui/raidboss/data/06-ew/trial/asura.ts

```diff
@@ -1,14 +1,197 @@
+import Outputs from '../../../../../resources/outputs';
+import { Responses } from '../../../../../resources/responses';
+import { DirectionOutputCardinal, Directions } from '../../../../../resources/util';
 import ZoneId from '../../../../../resources/zone_id';
 import { RaidbossData } from '../../../../../types/data';
 import { TriggerSet } from '../../../../../types/trigger';
 
-export type Data = RaidbossData;
+export interface Data extends RaidbossData {
+  seenFirstKhadga: boolean;
+}
+
+type Iconography = 'out' | 'in' | 'sides';
+
+const imageIconographyIds: { [id: string]: Iconography } = {
+  '8C82': 'out', // Pedestal Purge
+  '8C84': 'in', // Wheel of Deincarnation
+  '8C86': 'sides', // Bladewise
+};
+
+const outSafeSpots: Record<DirectionOutputCardinal, DirectionOutputCardinal> = {
+  'dirN': 'dirS',
+  'dirE': 'dirW',
+  'dirS': 'dirN',
+  'dirW': 'dirE',
+  'unknown': 'unknown',
+};
+
+const sidesSafeSpots: Record<DirectionOutputCardinal, DirectionOutputCardinal[]> = {
+  'dirN': ['dirE', 'dirW'],
+  'dirE': ['dirN', 'dirS'],
+  'dirS': ['dirE', 'dirW'],
+  'dirW': ['dirN', 'dirS'],
+  'unknown': ['unknown', 'unknown'],
+};
+
+const centerX = 100;
+const centerY = 100;
 
 const triggerSet: TriggerSet<Data> = {
   id: 'TheGildedAraya',
   zoneId: ZoneId.TheGildedAraya,
   timelineFile: 'asura.txt',
-  triggers: [],
+  initData: () => {
+    return {
+      seenFirstKhadga: false,
+    };
+  },
+  triggers: [
+    {
+      id: 'Asura Lower Realm',
+      type: 'StartsUsing',
+      netRegex: { id: '8CA1', source: 'Asura', capture: false },
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Asura Cutting Jewel',
+      type: 'StartsUsing',
+      netRegex: { id: '8CA0', source: 'Asura', capture: true },
+      response: Responses.tankBuster(),
```

Comment:
I know we didn’t test this directly, but because of the visual I’m comfortable just saying this should be `tankCleave()`.